src/pybind/mgr/rook/module.py

```diff
@@ -286,15 +274,12 @@ def serve(self):
             # in case we had a caller that wait()'ed on them long enough
             # to get persistence but not long enough to get completion
 
-            global all_completions
-            self.wait(all_completions)
-            all_completions = [c for c in all_completions if not c.is_complete]
+            self.wait(RookCompletionMixin.all_completions)
```

Comment:
Should this be `self.process(RookCompletionMixin.all_completions)`?

Reply:
this was also fixed one commit later

Reply:
same as above comment :-)